controllers/main.js

```diff
@@ -20,8 +21,7 @@ const remapDifficulty = {
 
 module.exports.playOrContinue = async (req, res) => {
   if (req.user?._id) {
```

Comment:
I guess later on instead of having checks like this, we could implement a route guard that is active for routes that need authentication